From c81ffcf2678f951fe31e8b2c800c7f704e4a73f8 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Wed, 6 May 2020 07:45:21 -0500
Subject: [PATCH 26/31] arm64: dts: beacon-imx8mm: Fix whitespace errors

There are extra tabs and extra newlines that are unnecessary.  This
patch fixes some of these whitespace issues.  There are no functional
changes.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 .../freescale/beacon-imx8mm-baseboard.dtsi    | 280 +++++++++---------
 1 file changed, 139 insertions(+), 141 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
index dcb2554711de..4b1202ff4e28 100644
--- a/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/beacon-imx8mm-baseboard.dtsi
@@ -164,165 +164,164 @@
 &iomuxc {
 	pinctrl-names = "default";
 
-	
-		pinctrl_csi_pwn: csi_pwn_grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_GPIO1_IO07_GPIO1_IO7		0x19
-			>;
-		};
+	pinctrl_csi_pwn: csi_pwn_grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_GPIO1_IO07_GPIO1_IO7		0x19
+		>;
+	};
 
-		pinctrl_csi_rst: csi_rst_grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_GPIO1_IO06_GPIO1_IO6		0x19
-				MX8MM_IOMUXC_GPIO1_IO14_CCMSRCGPCMIX_CLKO1	0x59
-			>;
-		};
+	pinctrl_csi_rst: csi_rst_grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_GPIO1_IO06_GPIO1_IO6		0x19
+			MX8MM_IOMUXC_GPIO1_IO14_CCMSRCGPCMIX_CLKO1	0x59
+		>;
+	};
 
-		pinctrl_i2c2: i2c2grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_I2C2_SCL_I2C2_SCL			0x400001c3
-				MX8MM_IOMUXC_I2C2_SDA_I2C2_SDA			0x400001c3
-			>;
-		};
+	pinctrl_i2c2: i2c2grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_I2C2_SCL_I2C2_SCL		0x400001c3
+			MX8MM_IOMUXC_I2C2_SDA_I2C2_SDA		0x400001c3
+		>;
+	};
 
-		/* Added for LogicPD Phoenix SOM*/
-		pinctrl_i2c4: i2c4grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_I2C4_SCL_I2C4_SCL			0x400001c3
-				MX8MM_IOMUXC_I2C4_SDA_I2C4_SDA			0x400001c3
-			>;
-		};
+	/* Added for LogicPD Phoenix SOM*/
+	pinctrl_i2c4: i2c4grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_I2C4_SCL_I2C4_SCL		0x400001c3
+			MX8MM_IOMUXC_I2C4_SDA_I2C4_SDA		0x400001c3
+		>;
+	};
 
-		pinctrl_espi2: espi2grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_ECSPI2_SCLK_ECSPI2_SCLK		0x82
-				MX8MM_IOMUXC_ECSPI2_MOSI_ECSPI2_MOSI		0x82
-				MX8MM_IOMUXC_ECSPI2_MISO_ECSPI2_MISO		0x82
-				MX8MM_IOMUXC_ECSPI2_SS0_ECSPI2_SS0		0x82
-			>;
-		};
+	pinctrl_espi2: espi2grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_ECSPI2_SCLK_ECSPI2_SCLK		0x82
+			MX8MM_IOMUXC_ECSPI2_MOSI_ECSPI2_MOSI		0x82
+			MX8MM_IOMUXC_ECSPI2_MISO_ECSPI2_MISO		0x82
+			MX8MM_IOMUXC_ECSPI2_SS0_ECSPI2_SS0		0x82
+		>;
+	};
 
-		pinctrl_led3: led3grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI3_RXFS_GPIO4_IO28	0x41
-			>;
-		};
+	pinctrl_led3: led3grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_SAI3_RXFS_GPIO4_IO28	0x41
+		>;
+	};
 
-		pinctrl_pcie0: pcie0grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_GPIO1_IO05_GPIO1_IO5	0x41
-				MX8MM_IOMUXC_SAI2_RXFS_GPIO4_IO21	0x41
-			>;
-		};
+	pinctrl_pcie0: pcie0grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_GPIO1_IO05_GPIO1_IO5	0x41
+			MX8MM_IOMUXC_SAI2_RXFS_GPIO4_IO21	0x41
+		>;
+	};
 
-		pinctrl_reg_usb_otg: reg_usb_otg {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI3_RXC_GPIO4_IO29     0x19
-			>;
-		};
+	pinctrl_reg_usb_otg: reg_usb_otg {
+		fsl,pins = <
+			MX8MM_IOMUXC_SAI3_RXC_GPIO4_IO29     0x19
+		>;
+	};
 
-		pinctrl_usbotg: usbotggrp {
-			fsl,pins = <
-				MX8MM_IOMUXC_GPIO1_IO10_USB1_OTG_ID     0x41
-			>;
-		};
+	pinctrl_usbotg: usbotggrp {
+		fsl,pins = <
+			MX8MM_IOMUXC_GPIO1_IO10_USB1_OTG_ID     0x41
+		>;
+	};
 
-		pinctrl_i2c4_gpio: i2c4grp-gpio {
-			fsl,pins = <
-				MX8MM_IOMUXC_I2C4_SCL_GPIO5_IO20        	0x1c3
-				MX8MM_IOMUXC_I2C4_SDA_GPIO5_IO21        	0x1c3
-			>;
-		};
+	pinctrl_i2c4_gpio: i2c4grp-gpio {
+		fsl,pins = <
+			MX8MM_IOMUXC_I2C4_SCL_GPIO5_IO20        	0x1c3
+			MX8MM_IOMUXC_I2C4_SDA_GPIO5_IO21        	0x1c3
+		>;
+	};
 
-		pinctrl_pcal6414: pcal6414-gpio {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI2_MCLK_GPIO4_IO27        	0x19
-			>;
-		};
+	pinctrl_pcal6414: pcal6414-gpio {
+		fsl,pins = <
+			MX8MM_IOMUXC_SAI2_MCLK_GPIO4_IO27        	0x19
+		>;
+	};
 
-		pinctrl_pdm: pdmgrp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI5_RXC_PDM_CLK		0xd6
-				MX8MM_IOMUXC_SAI5_RXD0_PDM_DATA0	0xd6
-			>;
-		};
+	pinctrl_pdm: pdmgrp {
+		fsl,pins = <
+			MX8MM_IOMUXC_SAI5_RXC_PDM_CLK		0xd6
+			MX8MM_IOMUXC_SAI5_RXD0_PDM_DATA0	0xd6
+		>;
+	};
 
-		pinctrl_sai3: sai3grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SAI3_TXFS_SAI3_TX_SYNC     0xd6
-				MX8MM_IOMUXC_SAI3_TXC_SAI3_TX_BCLK      0xd6
-				MX8MM_IOMUXC_SAI3_MCLK_SAI3_MCLK        0xd6
-				MX8MM_IOMUXC_SAI3_TXD_SAI3_TX_DATA0     0xd6
-				MX8MM_IOMUXC_SAI3_RXD_SAI3_RX_DATA0	0xd6
-			>;
-		};
+	pinctrl_sai3: sai3grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_SAI3_TXFS_SAI3_TX_SYNC     0xd6
+			MX8MM_IOMUXC_SAI3_TXC_SAI3_TX_BCLK      0xd6
+			MX8MM_IOMUXC_SAI3_MCLK_SAI3_MCLK        0xd6
+			MX8MM_IOMUXC_SAI3_TXD_SAI3_TX_DATA0     0xd6
+			MX8MM_IOMUXC_SAI3_RXD_SAI3_RX_DATA0	0xd6
+		>;
+	};
 
-		pinctrl_spdif1: spdif1grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SPDIF_TX_SPDIF1_OUT	0xd6
-				MX8MM_IOMUXC_SPDIF_RX_SPDIF1_IN		0xd6
-				MX8MM_IOMUXC_SPDIF_EXT_CLK_SPDIF1_EXT_CLK	0xd6
-			>;
-		};
+	pinctrl_spdif1: spdif1grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_SPDIF_TX_SPDIF1_OUT	0xd6
+			MX8MM_IOMUXC_SPDIF_RX_SPDIF1_IN		0xd6
+			MX8MM_IOMUXC_SPDIF_EXT_CLK_SPDIF1_EXT_CLK	0xd6
+		>;
+	};
 
-		pinctrl_uart2: uart2grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_UART2_RXD_UART2_DCE_RX	0x140
-				MX8MM_IOMUXC_UART2_TXD_UART2_DCE_TX	0x140
-			>;
-		};
+	pinctrl_uart2: uart2grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_UART2_RXD_UART2_DCE_RX	0x140
+			MX8MM_IOMUXC_UART2_TXD_UART2_DCE_TX	0x140
+		>;
+	};
 
-		pinctrl_uart3: uart3grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_ECSPI1_SCLK_UART3_DCE_RX		0x140
-				MX8MM_IOMUXC_ECSPI1_MOSI_UART3_DCE_TX		0x140
-				MX8MM_IOMUXC_ECSPI1_MISO_UART3_DCE_CTS_B	0x140
-				MX8MM_IOMUXC_ECSPI1_SS0_UART3_DCE_RTS_B		0x140
-			>;
-		};
+	pinctrl_uart3: uart3grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_ECSPI1_SCLK_UART3_DCE_RX		0x140
+			MX8MM_IOMUXC_ECSPI1_MOSI_UART3_DCE_TX		0x140
+			MX8MM_IOMUXC_ECSPI1_MISO_UART3_DCE_CTS_B	0x140
+			MX8MM_IOMUXC_ECSPI1_SS0_UART3_DCE_RTS_B	0x140
+		>;
+	};
 
-		pinctrl_usdhc2_gpio: usdhc2grpgpio {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CD_B_USDHC2_CD_B	0x41
-				MX8MM_IOMUXC_SD2_RESET_B_GPIO2_IO19	0x41
-			>;
-		};
+	pinctrl_usdhc2_gpio: usdhc2grpgpio {
+		fsl,pins = <
+			MX8MM_IOMUXC_SD2_CD_B_USDHC2_CD_B	0x41
+			MX8MM_IOMUXC_SD2_RESET_B_GPIO2_IO19	0x41
+		>;
+	};
 
-		pinctrl_usdhc2: usdhc2grp {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK		0x190
-				MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD		0x1d0
-				MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d0
-				MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d0
-				MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d0
-				MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d0
-				MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
-			>;
-		};
+	pinctrl_usdhc2: usdhc2grp {
+		fsl,pins = <
+			MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK	0x190
+			MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD	0x1d0
+			MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d0
+			MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d0
+			MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d0
+			MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d0
+			MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
+		>;
+	};
 
-		pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK		0x194
-				MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD		0x1d4
-				MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d4
-				MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d4
-				MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d4
-				MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d4
-				MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
-			>;
-		};
+	pinctrl_usdhc2_100mhz: usdhc2grp100mhz {
+		fsl,pins = <
+			MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK	0x194
+			MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD	0x1d4
+			MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d4
+			MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d4
+			MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d4
+			MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d4
+			MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
+		>;
+	};
 
-		pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
-			fsl,pins = <
-				MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK		0x196
-				MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD		0x1d6
-				MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d6
-				MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d6
-				MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d6
-				MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d6
-				MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
-			>;
-		};
+	pinctrl_usdhc2_200mhz: usdhc2grp200mhz {
+		fsl,pins = <
+			MX8MM_IOMUXC_SD2_CLK_USDHC2_CLK	0x196
+			MX8MM_IOMUXC_SD2_CMD_USDHC2_CMD	0x1d6
+			MX8MM_IOMUXC_SD2_DATA0_USDHC2_DATA0	0x1d6
+			MX8MM_IOMUXC_SD2_DATA1_USDHC2_DATA1	0x1d6
+			MX8MM_IOMUXC_SD2_DATA2_USDHC2_DATA2	0x1d6
+			MX8MM_IOMUXC_SD2_DATA3_USDHC2_DATA3	0x1d6
+			MX8MM_IOMUXC_GPIO1_IO04_USDHC2_VSELECT	0x1d0
+		>;
+	};
 };
 
 &csi1_bridge {
@@ -416,7 +415,6 @@
 };
 
 &i2c4 {
-
 	clock-frequency = <400000>;
 	pinctrl-names = "default", "gpio";
 	pinctrl-0 = <&pinctrl_i2c4>;
-- 
2.17.1

